shared/src/store/action-reducers/transfer.ts

```diff
@@ -143,14 +151,21 @@ export namespace TransferActionReducers {
                 duration = startPoint.duration;
             }
 
+            // Remove the position of the element
+            if (elementType === 'personnel') {
+                removeElementPosition(draftState, 'personnel', element.id);
+            } else {
+                element.position = undefined;
+            }
+
             // Set the element to transfer
-            delete element.position;
             element.transfer = {
```

Comment:
Isn't there a difference between `element.position = undefined` (new) and `delete element.position` (old)?

Reply:
Using `delete` also removes the key whereas setting it to `undefined` only changes the value.
I would expect the key to still be there but just the value set to undefined.

To remove those concerns and also to avoid having different objects after serialisation (`Json.parse(Json.stringify({ a: undefined}))`)  we should replace all occasions of `undefined` in the models with `null`. But this should be done at a later point in time.

Reply:
Right. So the new behavior is better than the old one? Or is it good (or bad) either way?

Reply:
I would say it is a bit better but still not great.

Reply:
Ok, so we can leave it as it is.